k8s-deployment.yaml

```
@@ -58,10 +58,12 @@ spec:
             - containerPort: 80
           env:
             - name: DEFAULT_NODE_NAME
-              value: "mocklet"
+              valueFrom:
+                fieldFrom:
```

From https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/ should this be `fieldRef` instead?
```suggestion
                fieldRef:
```

---

Yes. No idea how that happened...